spec/acceptance/00_backend_spec.rb

```diff
@@ -13,14 +13,75 @@
       apply_manifest_on(node, pp, :catch_changes  => true)
     end
 
-    it do
-      on node, "/opt/puppetlabs/bin/facter os" do
-        puts stdout
-      end
+    describe service('sensu-backend'), :node => node do
+      it { should be_enabled }
+      it { should be_running }
+    end
+  end
+
+  context 'with SSL' do
+    it 'should bootstrap certs' do
+      on node, 'mkdir -p /root/ssl'
+      on node, 'openssl req -newkey rsa:2048 -nodes -keyout /root/ssl/ca_key.pem -x509 -days 365 -out /root/ssl/ca.pem -subj "/CN=localhost"'
+      on node, 'openssl req -newkey rsa:2048 -nodes -keyout /root/ssl/key.pem -out /root/ssl/localhost.csr -subj "/CN=localhost"'
+      on node, 'openssl x509 -req -in /root/ssl/localhost.csr -CA /root/ssl/ca.pem -CAkey /root/ssl/ca_key.pem -CAcreateserial -out /root/ssl/cert.pem -days 365 -sha256'
+      on node, 'cat /root/ssl/ca.pem >> /etc/pki/tls/certs/ca-bundle.crt'
```

Comment:
From the acceptance tests failures it would seem that this works differently with Ubuntu.